transip_rest_client/generic_rest_client.py

```diff
@@ -115,36 +104,9 @@ def _do_request(self,
             extra_headers = {}
         url = urljoin(self.base_url, endpoint)
         try:
-            if call_type == 'get':
-                response = requests.get(
-                    url,
-                    params=params,
```

Comment:
here the call is params=params

Reply:
ahh right, fixed in 65f24a58ca69a520e70331acc9842bf897017a74